packages/fxa-react/components/AppErrorDialog/index.scss

```diff
@@ -0,0 +1,10 @@
+.generic-error {
+  max-width: 360px;
+  text-align: center;
+
+  h2 {
+    font-size: 22px;
+    font-weight: 700;
+    margin-bottom: 10px;
+  }
+}
```

Comment:
🙃 We should discuss tomorrow. I left a note in my draft PR (#5460) around "Figure out what we're doing about shared classes in `fxa-react`".

I think we'd said worst case scenario we generally _don't_ want an external stylesheet in `fxa-react` though, I think it makes sense for the spinner that those will be the same across the board but these feel off / like they can be passed in as classes.